tardis/io/decay.py

```diff
@@ -86,4 +86,33 @@ def decay(self, t):
         
         df = IsotopeAbundances.from_materials(decayed_materials)
         df.sort_index(inplace=True)
-        return df 
\ No newline at end of file
+        return df 
+
+    def as_atomic_numbers(self, abundance, t, normalize=True):
```

Comment:
thi should be called - `merge_isotopes` and should only perform the groupby operation.